forms-flow-web/src/routes/Design/Process/index.js

```diff
@@ -14,7 +14,7 @@ const DesignerProcessRoute = ({ component: Component, ...rest }) => (
   <Route
     {...rest}
     render={(props) => {
-      if (user.includes('create_designs')) {
+      if ((user.includes('create_designs') || user.includes('view_designs'))) {
```

Comment:
could you please use some instead of includes because if the user not includes first permission then the same time will take to check second permission 

user.some((i)=> i == create_designs || i == view_designes